features/telemetry/protocols/bmp/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/adapter/BmpPeerStatusAdapter.java

```diff
@@ -77,13 +74,18 @@ public BmpPeerStatusAdapter(final AdapterDefinition adapterConfig,
     public void handleMessage(final TelemetryMessageLogEntry message,
                               final TelemetryMessageLog messageLog) {
         LOG.trace("Parsing packet: {}", message);
-        final BsonDocument document = new RawBsonDocument(message.getByteArray());
+        final Transport.Message document;
```

Comment:
Should call this something other than `document` now?

Reply:
There is no good name I can come up with - `message` is already in use.